spec/finders/conversation_finder_spec.rb

```diff
@@ -160,9 +160,15 @@
       let(:params) { { status: 'open', assignee_type: 'me', conversation_type: 'unattended' } }
 
       it 'returns unattended conversations' do
-        create_list(:conversation, 25, account: account, inbox: inbox, assignee: user_1)
+        create(:conversation, account: account, first_reply_created_at: Time.now.utc, assignee: user_1) # attended_conversation
+        # to ensure that waiting since value is populated
+        # create(:message, message_type: :outgoing, conversation: attended_conversation, account: account)
+
+        create(:conversation, account: account, first_reply_created_at: nil, assignee: user_1) # unattended_conversation_no_first_reply
+        create(:conversation, account: account, first_reply_created_at: Time.now.utc, assignee: user_1) # unattended_conversation_waiting_since
```

Comment:
Should we create an incoming message here for this conversation to stimulate the real-life case or populate a value for waiting since ?